.gitignore

```diff
@@ -1,5 +1,4 @@
 # Build Folders (you can keep bin if you'd like, to store dlls and pdbs)
-[Bb]in/
```

Comment:
I'll just exclude top level /bin dirs then? Or revert the gitignore  change and force add the node bin files?